### PR TITLE
add post-Meetup survey template

### PR DESCRIPTION
**Pull request by kariannemah**

### CODE REVIEW

File: survey/post-event-survey.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  admin_only: false
+  type: ResponseFieldEmail
+rating_fields: []
+labels:
+- name: Missing information
+  color: 3600ff
+- name: Questionable
+  color: '336699'
+statuses:
+- name: Reviewed by co-organizers
```
Comment (ajb) [inline survey/post-event-survey.yml]: Wouldn't it be an issue if this is the only/default status? e.g. how could an application be "reviewed by co-organizers" immediately after it was received?

**Comment by ajb** [comment]
:+1: other than the one comment above

File: survey/post-event-survey.yml (hunk 1 above)
Comment (kariannemah) [inline survey/post-event-survey.yml]: Indeed. I added a default status of "open for review" in https://github.com/dobtco/screendoor-project-templates/commit/76380e41298a6882ef865908b18a4a5d2c5df742.

**Comment by jrubenoff** [comment]
Nice!

Just to clarify: these are examples for the template library, right? Are they preloaded for new users?

**Comment by kariannemah** [comment]
Yup, these examples live here, for now. The only templates in this repo that are preloaded are the ones in `default`.

**Comment by jrubenoff** [comment]
Cool, just want to avoid adding too many templates to the first-run experience.

**Comment by ajb** [comment]
A clarifying note, too -- the templates in `default` are duplicated inside
of the Screendoor repository -- we haven't taken the time to design a
system that will let us "pull in" the templates from this repo.

On Mon, Feb 2, 2015 at 5:09 PM, Josh Rubenoff notifications@github.com
wrote:

> Cool, just want to avoid adding too many templates to the first-run
> experience.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dobtco/screendoor-project-templates/pull/4#issuecomment-72548802
> .

## 

Adam Becker
(951) 9-BECKER
@AdamJacobBecker

**Comment by kariannemah** [comment]
Right. In the README, I'll explain that the `default` templates are the same projects from which a user can choose when she clicks "New project..."

**Pull request merged**